File: Unidad 4/C#/Prog_18_Uni_4.cs
﻿using System;

namespace Prog_18_Uni_4 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int[,] diag = { { 1, 2, 3 }, { 4, 5, 6 }, { 7, 8, 9 } };

            Console.WriteLine("Diagonal principal: ");
            for (int c = 0; c < diag.GetLength(0); c++)
            {
                Console.WriteLine(diag[c, c] + " ");
            }

            Console.WriteLine("Diagonal inversa:  ");
            for (int i = 0; i < 3; i++)
            {
                Console.WriteLine(diag[i, diag.GetLength(1) - 1 - i]);
            }

            Console.WriteLine("Diagonal secundaria:  ");
            for (int i = 0; i < 2; i++)
            {
                Console.WriteLine(diag[i, i + 1] + " ");
            }
        }
    }
}

File: Unidad 4/Java/Prog_18_Uni_4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.prog_18_uni_4;

/**
 *
 * @author USUARIO LENOVO
 */
public class Prog_18_Uni_4 {

    public static void main(String[] args) {
         int diag[][] = { {1,2,3} , {4,5,6} , {7,8,9} };
            System.out.println("Diagonal principal: ");
            for (int c=0; c<diag.length; c++) {
                System.out.println(diag[c][c]+" ");
            }
            System.out.println("Diagonal inversa:  ");
            for(int i=0; i<3; i++) {
               System.out.println(diag[i][diag.length -1 -i]); 
               {
            }
            }
             System.out.println("Diagonal secundaria:  ");
            for(int i=0; i<2; i++)
               System.out.println(diag[i][i+1]+" "); 
    }
}
